<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWin</class>
 <widget class="QDialog" name="mainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Package Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QTabWidget" name="tabUpdates">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="systab">
      <attribute name="title">
       <string>Add / Remove Packages</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupUpdates">
         <property name="title">
          <string>System Packages</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QRadioButton" name="radioBasic">
              <property name="text">
               <string>Basic</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioAdvanced">
              <property name="text">
               <string>Advanced</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="spacer2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>223</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushPkgApply">
              <property name="text">
               <string>&amp;Apply</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QStackedWidget" name="stackedWidget">
            <widget class="QWidget" name="pageBasic">
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QListWidget" name="listWidgetBasic"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="pageAdv">
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <widget class="QListWidget" name="listWidgetAdv"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pkgtab">
      <attribute name="title">
       <string>Package Updates</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupUpdatesPkgs">
         <property name="title">
          <string>Available Updates</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="buttonRescanPkgs">
              <property name="text">
               <string>Re&amp;scan packages</string>
              </property>
              <property name="icon">
               <iconset resource="pc-pkgmanager.qrc">
                <normaloff>:/view-refresh.png</normaloff>:/view-refresh.png</iconset>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <spacer name="spacer2_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>223</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushUpdatePkgs">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>&amp;Update packages</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QTreeWidget" name="listViewUpdatesPkgs">
            <property name="indentation">
             <number>0</number>
            </property>
            <attribute name="headerCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="headerDefaultSectionSize">
             <number>160</number>
            </attribute>
            <column>
             <property name="text">
              <string>Package</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Old Version</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>New Version</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QProgressBar" name="progressUpdate">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="textStatus">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Installing updates</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pc-pkgmanager.qrc"/>
 </resources>
 <connections/>
</ui>
